src/connectedGraph.js

```diff
@@ -0,0 +1,15 @@
+import setCompliment from '../src/setCompliment'
+export default function connectedGraph(graph){
+  let allPoints = Object.keys(graph)
+  let connected = []
+  let toCheck = [allPoints[0]]
+
+  while(toCheck.length>0){
+    let neighbors = graph[toCheck[0]]
+    neighbors.map((points) => toCheck.push(points))
```

Comment:
shouldn't this be 
```js
neighbors.map((point) => toCheck.push(point))
```
`point` instead of `points` since the value is a single character string as apposed to say an array of strings?